tests/test_extensions/test_sklearn_extension/test_sklearn_extension.py

```diff
@@ -1804,7 +1804,6 @@ def test_run_on_model_with_empty_steps(self):
         clf = sklearn.pipeline.Pipeline([
             ('dummystep', 'passthrough'),  # adding 'passthrough' as an estimator
             ('prep', clf),
-            ('variancethreshold', None),  # adding 'None' as an estimator
```

Comment:
Is there a reason you dropped this and didn't replace it by `drop`?

Reply:
Yeah, since Pipeline supports only _None_ and _passthrough_.